app/src/main/kotlin/com/wire/android/ui/home/settings/backup/BackupAndRestoreViewModel.kt

```diff
@@ -232,7 +232,11 @@ class BackupAndRestoreViewModel
             backupRestoreProgress = BackupRestoreProgress.InProgress(),
             restorePasswordValidation = PasswordValidation.NotVerified
         )
-        if (kaliumFileSystem.exists(latestImportedBackupTempPath)) kaliumFileSystem.delete(latestImportedBackupTempPath)
+        if (this@BackupAndRestoreViewModel::latestImportedBackupTempPath.isInitialized
+            && kaliumFileSystem.exists(latestImportedBackupTempPath)
+        ) {
+            kaliumFileSystem.delete(latestImportedBackupTempPath)
+        }
```

Comment:
```suggestion
        if (this@BackupAndRestoreViewModel::latestImportedBackupTempPath.isInitialized &&
            kaliumFileSystem.exists(latestImportedBackupTempPath)) {
            kaliumFileSystem.delete(latestImportedBackupTempPath)
        }
```

Reply:
In my IDE it's reformated to my changes 😅